src/fann_io.c

```diff
@@ -181,6 +181,7 @@ int fann_save_internal_fd(struct fann *ann, FILE * conf, const char *configurati
 	fprintf(conf, "network_type=%u\n", ann->network_type);
 	
 	fprintf(conf, "learning_momentum=%f\n", ann->learning_momentum);
+	fprintf(conf, "learning_l2_norm=%f\n", ann->learning_l2_norm);
```

Comment:
Are there any versioning concerns here or are we free to break reading of ANN files that have been written with an older library version?